pkg/cmd/backuprestore/backupserver.go

```diff
@@ -0,0 +1,142 @@
+package backuprestore
+
+import (
+	"context"
+	"errors"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"k8s.io/klog/v2"
+
+	"github.com/adhocore/gronx/pkg/tasker"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+const (
+	backupVolume = "/var/backup/etcd/"
+	backupFolder = "current-backup"
```

Comment:
@tjungblu  i needed to add this subfolder, since https://github.com/Elbehery/cluster-etcd-operator/blob/00a3a5ca4cb300054e557193b5fc8183493ebabc/pkg/cmd/backuprestore/fileioutils.go#L73 was failing to remove the backup folder (i.e. /var/backup/etcd)

if this hack is not ideal, we need to update https://github.com/Elbehery/cluster-etcd-operator/blob/00a3a5ca4cb300054e557193b5fc8183493ebabc/pkg/cmd/backuprestore/fileioutils.go#L73 not to fail upon deleting the dir if exists